### PR TITLE
Create DID screen

### PR DESCRIPTION
**Pull request by osinakayah**

### CODE REVIEW

File: src/features/didManagement/CreateNewDIDScreen.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+          <NBox width="80px" alignItems="flex-end" />
+        </Box>
+      </Header>
+      <ScrollView marginLeft={5} marginRight={5}>
+        <FormControl>
+          <Stack mt={7}>
+            <FormControl.Label>
+              {translate('didManagement.did_name')}
+            </FormControl.Label>
+            <Input
```
Comment (maycon-mello) [inline src/features/didManagement/CreateNewDIDScreen.js:59]: Can we add testIds on this screen?

File: src/features/didManagement/CreateNewDIDScreen.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+          <Typography my={2} color={Theme.colors.inactiveText}>
+            {translate('didManagement.learn_more') + ' '}
+            <Icon as={Ionicons} name="open-outline" size="xs" />
+          </Typography>
+        </VStack>
+        <DIDAdvancedOptions onChange={onChange} form={{}} />
+        <LoadingButton
+          full
+          testID="next-btn"
```
Comment (maycon-mello) [inline src/features/didManagement/CreateNewDIDScreen.js:112]: Shouldn't we use that helper to generate testIds?

File: src/features/didManagement/EditDIDScreen.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+              onChangeText={onChange('word1')}
+              autoCapitalize="none"
+            />
+          </Stack>
+        </FormControl>
+      </ScrollView>
+      <NBox mx={7}>
+        <LoadingButton
+          full
+          testID="save-btn"
```
Comment (maycon-mello) [inline src/features/didManagement/EditDIDScreen.js:58]: Same question about the helper to generate testIds

File: src/features/didManagement/EditDIDScreen.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  ScreenContainer,
+  Typography,
+} from '../../design-system';
+import {navigateBack} from '../../core/navigation';
+import {translate} from '../../locales';
+import {FormControl, ScrollView, Stack} from 'native-base';
+export function EditDIDScreen() {
+  const onChange = useCallback(() => {}, []);
+  return (
+    <ScreenContainer testID="CreateNewDIDScreen">
```
[the remark this answers is not in the document]
Comment (maycon-mello) [reply src/features/didManagement/EditDIDScreen.js:18]: `{...addTestId('CreateNewDIDScreen')}`

File: src/features/didManagement/CreateNewDIDScreen.js [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+  Select,
+  Stack,
+  VStack,
+} from 'native-base';
+import {translate} from '../../locales';
+import {DIDAdvancedOptions} from './components/DIDAdvancedOptions';
+import {Ionicons} from '@native-base/icons';
+
+export function CreateNewDIDScreen() {
+  const onChange = useCallback(() => {}, []);
```
Comment (maycon-mello) [inline src/features/didManagement/CreateNewDIDScreen.js:28]: Shouldn't we have an integration with the `wallet-sdk` on it?

Comment (osinakayah) [reply src/features/didManagement/CreateNewDIDScreen.js:28]: For now, it is just a stub, I am yet to implement it

File: src/features/didManagement/CreateNewDIDScreen.js [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+            </FormControl.HelperText>
+          </Stack>
+        </FormControl>
+        <FormControl>
+          <Stack mt={7}>
+            <FormControl.Label>
+              {translate('didManagement.did_type')}
+            </FormControl.Label>
+            <Select>
+              <Select.Item label="did:key" value="didkey" />
```
[the remark this answers is not in the document]
Comment (osinakayah) [reply src/features/didManagement/CreateNewDIDScreen.js:80]: Yes, Rey implemented it in the UI, there is a modal that pops up, telling the user, they will have to pay for did:dock if the select did:dock from the dropdown.

**Review by maycon-mello — APPROVED** [review]

**Pull request merged**